src/js/components/stock-movement-wizard/inboundV2/InboundV2.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useMemo } from 'react';
+
+import Button from 'components/form-elements/Button';
+import InboundHeader from 'components/stock-movement-wizard/inboundV2/InboundHeader';
+import InboundV2AddItems
+  from 'components/stock-movement-wizard/inboundV2/sections/InboundV2AddItems';
+import InboundV2Create from 'components/stock-movement-wizard/inboundV2/sections/InboundV2Create';
+import InboundV2Send from 'components/stock-movement-wizard/inboundV2/sections/InboundV2Send';
+import WizardStepsV2 from 'components/wizard/v2/WizardStepsV2';
+import inboundV2Step from 'consts/InboundV2Step';
+import mockInboundV2AdditionalTitle from 'consts/MockInboundV2AdditionalTitle';
+import mockInboundV2Title from 'consts/MockInboundV2Title';
+import useTranslate from 'hooks/useTranslate';
+import useTranslation from 'hooks/useTranslation';
+import useWizard from 'hooks/useWizard';
+import PageWrapper from 'wrappers/PageWrapper';
+
+const InboundV2 = () => {
+  useTranslation('stockMovement');
+  const translate = useTranslate();
+
+  const steps = useMemo(() => [
+    {
+      key: inboundV2Step.CREATE,
+      title: translate('react.stockMovement.create.label', 'Create'),
+      Component: () => (<InboundV2Create />),
+    },
+    {
+      key: inboundV2Step.ADD_ITEMS,
+      title: translate('react.stockMovement.addItems.label', 'Add Items'),
+      Component: () => (<InboundV2AddItems />),
+    },
+    {
+      key: inboundV2Step.SEND,
+      title: translate('react.stockMovement.send.label', 'Send'),
+      Component: () => (<InboundV2Send />),
+    },
+  ], [translate]);
+
+  const stepsTitles = steps.map((step) => ({
+    title: step.title,
+    key: step.key,
+  }));
+
+  const [
+    Step,
+    {
+      // navigateToStep,
+      next,
+      previous,
+      is,
+    },
+  ] = useWizard({
+    initialKey: InboundV2.DETAILS,
+    steps,
+  });
+
+  // this will still need to be improved in the future
+  const headerAdditionalTitle = is(inboundV2Step.SEND) ? mockInboundV2AdditionalTitle : undefined;
+  const headerTitle = is(inboundV2Step.ADD_ITEMS)
+  || is(inboundV2Step.SEND) ? mockInboundV2Title : undefined;
+
+  return (
+    <PageWrapper>
+      <WizardStepsV2 steps={stepsTitles} currentStepKey={Step.key} />
+      <InboundHeader
+        title={headerTitle}
+        additionalTitle={headerAdditionalTitle}
+      />
+      {/* <form> */}
+      {is(inboundV2Step.CREATE) && (<Step.Component />)}
+      {is(inboundV2Step.ADD_ITEMS) && (<Step.Component />)}
+      {is(inboundV2Step.SEND) && (<Step.Component />)}
+
+      {/* these buttons are for testing purposes now */}
+      <div className="d-flex justify-content-between">
+        <Button
+          label="react.stockMovement.form.previous.label"
+          defaultLabel="Previous"
+          variant="primary"
+          className="fit-content align-self-end"
+          onClick={previous}
```

Comment:
this doesn't change anything here, but be aware that by calling it like that, if the `previous` was expecting to receive any argument, this would be run with the `event` argument.
What you did is equivalent to do:
```js
onClick={e => previous(e)}
```
so when you want to make sure you call a method without any params here, you should do:
```js
onClick={() => previous()}
```
but as I said, this doesn't change anything for this particular method. Treat it as mentoring comment.